packages/wonder-blocks-tooltip/src/components/tooltip.tsx

```diff
@@ -210,6 +228,8 @@ export default class Tooltip extends React.Component<Props, State> {
                     <TooltipBubble
                         id={bubbleId}
                         style={props.style}
+                        contentStyle={contentStyle}
+                        backgroundColor={backgroundColor}
```

Comment:
Hmm... It looks like the border radius and maybe drop shadow might be getting overridden with the custom styles? I'm not entirely sure off the top of my head why this could be happening. Let me know if you'd like help investigating!

<img width="1115" alt="image" src="https://github.com/Khan/wonder-blocks/assets/13231763/f9e2a59a-3441-4dfc-8600-d5cd3f89d30b">

Reply:
good catch! i was changing the background color on the wrong component. it should have been on an inner view. this actually simplifies this a bit. thanks nisha :)